JavaScript/JavaScriptSDK.Tests/CheckinTests/appInsights.tests.ts

```diff
@@ -1454,8 +1454,10 @@ class AppInsightsTests extends TestClass {
             test: () => {
                 var appInsights = new Microsoft.ApplicationInsights.AppInsights(this.getAppInsightsSnippet());
                 var trackStub = this.sandbox.stub(appInsights.context, "track");
-                var name = "test";
-                var url = "http://myurl.com";
+                var pathName = "https://tempurl.net/api/temp/ABCD";
+                var commandName = "GET https://tempurl.net/api/temp/ABCD?param1=test&param2=test";
+                var url = "https://tempurl.net/api/temp/ABCD?param1=test&param2=test";
```

Comment:
It doesn't really matter but I would probably have built url as pathName + the param string and then build commandName as "GET " + url so there would only be one copy of the string to type/maintain.